Frameworks/UIKit/UIFont.mm

```diff
@@ -856,102 +321,33 @@ - (instancetype)copyWithZone:(NSZone*)zone {
     return [self retain];
 }
 
-/**
- @Status Interoperable
-*/
-- (void)dealloc {
-    _fileName = nil;
-    _name = nil;
-    _descriptor = nil;
-    [super dealloc];
-}
-
 - (void)_setName:(NSString*)name {
-    _fileName = name;
+    UNIMPLEMENTED();
 }
 
 - (uint32_t)_fontHandle {
-    return (uint32_t)_font;
+    UNIMPLEMENTED();
+    return StubReturn();
 }
 
 - (uint32_t)_sizingFontHandle {
-    return (uint32_t)_sizingFont;
+    UNIMPLEMENTED();
+    return StubReturn();
 }
 
 // Internal methods
 
 // Private message sent from CTFontManager for the implementation of CTFontManagerRegisterGraphicsFont
 - (bool)_CTFontManagerRegisterGraphicsFont:(CGFontRef)font withError:(CFErrorRef*)error {
```

Comment:
I think I'd rather do that in a separate CR? For example, for this function, it probably makes more sense to do the full stripping at the same time as bringing up CTFontManager. Doing it in the current state would probably generate a lot of unnecessary extra work just to keep things building.

---
In reply to: [78222243](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78222243) [](ancestors = 78222243)